Cargo.toml

```diff
@@ -55,6 +55,7 @@ rustls = { version = "0.20", optional = true }
 rustls-pemfile = { version = "1.0", optional = true }
 socket2 = "0.4"
 get_if_addrs = "0.5"
+markdown = "0.3.0"
```

Comment:
Perhaps [comrak](https://crates.io/crates/comrak) would be more suitable for the fancier markdowns that people use nowadays?

Reply:
Yeah, I think this is better than the one I was using. I'll switch it.

Reply:
I changed it. Thought comrak seems to be a bit heavy while markdown was lightweight. It does support the html bits and has the same look as github one so I think it's ok.